lib/web/apiserver.go

```diff
@@ -2128,7 +2130,12 @@ func (h *Handler) siteSessionsGet(w http.ResponseWriter, r *http.Request, p http
 
 	sessions := make([]session.Session, 0, len(trackers))
 	for _, tracker := range trackers {
-		if tracker.GetSessionKind() == types.SSHSessionKind && tracker.GetState() != types.SessionState_SessionStateTerminated {
+		// Only return ssh and k8s session type.
+		if tracker.GetSessionKind() != types.SSHSessionKind && tracker.GetSessionKind() != types.KubernetesSessionKind {
+			continue
+		}
+
+		if tracker.GetState() != types.SessionState_SessionStateTerminated {
```

Comment:
Would it make more sense to assign the kind to a variable above to make this easier to read? That may also make it compact enough to move the state check into this skip conditional.